error.go

```diff
@@ -61,7 +61,7 @@ func (e ErrCode) rawMessage() string {
 	}
 }
 
-type meilisearchApiError struct {
```

Comment:
If I'm correct this has nothing to do with the test PR, can we put it in another PR please?

Reply:
No, Because this change is dependent on this PR being made and being public.

Reply:
meilisearchApiError used internally but for integration tests need to changed public.